src/MusicStore/Controllers/AccountController.cs

```diff
@@ -151,8 +153,10 @@ public IActionResult Register()
                     // For more information on how to enable account confirmation and password reset please visit http://go.microsoft.com/fwlink/?LinkID=320771
                     // Send an email with this link
                     string code = await UserManager.GenerateEmailConfirmationTokenAsync(user);
-                    var callbackUrl = Url.Action("ConfirmEmail", "Account", new { userId = user.Id, code = code }, protocol: HttpContext.Request.Scheme);
-                    await MessageServices.SendEmailAsync(model.Email, "Confirm your account",
+                    var callbackUrl = Url.Action(
+                        "ConfirmEmail", "Account", new { userId = user.Id, code = code }, protocol: HttpContext.Request.Scheme);
+                    await MessageServices.SendEmailAsync(
+                        model.Email, "Confirm your account",
                         "Please confirm your account by clicking this link: <a href=\"" + callbackUrl + "\">link</a>");
```

Comment:
Style is generally "if you wrap, give every argument its own line." [Line 159](https://github.com/aspnet/MusicStore/pull/605/files#diff-d9ee8fb38dcc504d9616e8e6611fae7cR159) looks particularly odd.